test/adapter/xml/test_xml_deserialization.py

```diff
@@ -177,7 +177,7 @@ def test_reference_kind_mismatch(self) -> None:
         with self.assertLogs(logging.getLogger(), level=logging.WARNING) as context:
             read_aas_xml_file(io.BytesIO(xml.encode("utf-8")), failsafe=False)
         for s in ("GLOBAL_REFERENCE", "IRI=http://acplt.org/test_ref", "Asset"):
-            self.assertIn(s, context.output[0])  # type: ignore
+            self.assertIn(s, context.output[0])
```

Comment:
I think this wasn't merged correctly, as this change isn't part of the resulting merge. Can you check this again?

Reply:
Fixed in 6d58ca7